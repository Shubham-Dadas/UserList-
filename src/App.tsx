import * as React from "react";
import UsersList from "./components/pages/UserList/UserList";

function App() {
  return (
    <div className="App">
      <UsersList />
    </div>
  );
}

export default App;

