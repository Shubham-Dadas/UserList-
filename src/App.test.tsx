import React from "react";
import toJson from "enzyme-to-json";
import { shallow, mount } from "enzyme";
import App from "./App";

describe("App component", () => {
  it("test component", () => {
    const component = mount(<App />);
    // @ts-ignore
    expect(toJson(component)).toMatchSnapshot();
    expect(1 + 2).toBe(3);
  });
});
