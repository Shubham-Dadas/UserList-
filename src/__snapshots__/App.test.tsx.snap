// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component test component 1`] = `
<App>
  <div
    className="App"
  >
    <UsersList>
      <div
        className="user-list-container"
      >
        <div
          className="loading"
        >
          <p>
            Loading users...
          </p>
        </div>
      </div>
    </UsersList>
  </div>
</App>
`;
